common-lib/src/main/resources/application.conf

```diff
@@ -4,9 +4,12 @@ play {
   application.loader = "AppLoader"
   application.langs = "en"
 
-  http.session {
-    httpOnly = false
-    secure = true
+  http{
+    session {
+      httpOnly = false
+      secure = true
+    }
+    forwarded.trustedProxies=["0.0.0.0/0", "::/0"]
```

Comment:
On our branch, the new dev-nginx was enough to get POSTs working through the dashboard; we didn't need this extra config. It might not be needed?

Reply:
ahh good to know, this is still needed for it to work in AWS (behind the load balancer, see https://www.playframework.com/documentation/2.8.x/HTTPServer#Configuring-trusted-proxies)